string arm_name
string object_name


